mypy outputs error reports for "not just the current file"

Example output, running on `prettify.py`:

```
prettify.py:42: error: No library stub file for module 'yaml'
prettify.py:42: note: (Stub files are from https://github.com/python/typeshed)
prettify.py:11: note: In module imported here:
_common.py: note: In function "read_archive_lines":
_common.py:7: error: "module" has no attribute "open" 
```

There is a command argument `--silent-imports`, but using that will silence type-checking of _all_ imported things, which is undesirable.

The best idea is probably to provide a `PythonLinter.module` attribute and then interface mypy directly in the `PythonLinter.check()` method.

--- a/linter.py
+++ b/linter.py
@@ -18,12 +18,13 @@ class Mypy(PythonLinter):
     """Provides an interface to mypy."""
 
     syntax = 'python'
-    cmd = 'mypy * @'
+    cmd = 'mypy --follow-imports silent * @'
     version_args = '--version'
     version_re = r'(?P<version>\d+\.\d+)'
     version_requirement = '>= 0.2'
     regex = r'^.+\.py:(?P<line>\d+): (?:error|warning|note): (?P<message>.+)'
     multiline = False
+    template_suffix = '-'
     line_col_base = (1, 1)
     tempfile_suffix = None
     error_stream = util.STREAM_BOTH

